src/localization/locale/en.ts

```diff
@@ -87,7 +88,7 @@ export const en = {
   },
   info: {
     artifact: 'Artifact',
-    noResults: 'No tracked results',
+    noResults: 'No tracked results, enable [result persistence](https://docs.prefect.io/concepts/results/#persisting-results) to track results.',
```

Comment:
This only affects old client versions basically right?

Reply:
Won't they need to upgrade to a version with result artifacts? Maybe we should just suggest being on the latest version.

Reply:
Afaik, yeah. It'll show up for empty states for task runs or the flow run itself as well which at the very least is a helpful "what this feature is"